pkg/controller/eventhandler.go

```diff
@@ -1,13 +1,12 @@
 package controller
 
 import (
-	shipper "github.com/bookingcom/shipper/pkg/apis/shipper/v1alpha1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/client-go/tools/cache"
 	"k8s.io/klog"
 )
 
-func NewAppClusterEventHandler(callback func(obj interface{})) cache.ResourceEventHandler {
+func NewAppClusterEventHandler(filterLabel string, callback func(obj interface{})) cache.ResourceEventHandler {
```

Comment:
Shall we start thinking about introducing a more abstract Filter substance here that can carry multiple labels etc?

Reply:
Yes and no. I think I got the abstraction wrong here in the first place. I pushed 35d7f2d, which is probably not what you meant, but I think it addresses the issue anyway. What do you think?